src/chargen.cpp

```diff
@@ -939,6 +941,9 @@ void create_parse(struct descriptor_data *d, const char *arg)
       obj_to_char(read_object(2041, VIRTUAL), d->character);
     GET_EQ(d->character, WEAR_BODY) = read_object(shirts[number(0, 4)], VIRTUAL);
     GET_EQ(d->character, WEAR_LEGS) = read_object(pants[number(0, 4)], VIRTUAL);
+    GET_EQ(d->character, WEAR_FEET) = read_object(shoes[number(0, 4)], VIRTUAL);
+    if (real_object(1815) > -1)
+      obj_to_char(read_object(1815, VIRTUAL), d->character);
```

Comment:
I think it's a good idea to have this here, but to complete this change we need to do several things:

- Set the radio to 8
- Make Chargen !RADIO
- Remove the radio-granting code from spec_procs.cpp

I'm thinking setting the channel and taking out the grant code is reasonable for this PR, what do you think?